### PR TITLE
Linkall without losing color

### PR DESCRIPTION
**Pull request by stephanwlee**
spawn allows you to mock the io like ansi so `npm` doesn't print in no-color.

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by yahoocla** [comment]
CLA is valid!

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "unit": "tap test/*.js",
     "unit-file": "tap",
     "linkall": "node ./scripts/linkall.js",
+    "unlinkall": "find . -name a -type d -exec rm -rf {} \\;",
```
Comment (irae) [inline package.json]: ???

Comment (stephanwlee) [inline package.json]: :\ I felt like there should be the opposite of linkall

File: packages/mendel-cli/package.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "name": "mendel",
   "version": "1.3.0",
   "description": "Mendel command line utility",
+  "preferGlobal": true,
```
Comment (irae) [inline packages/mendel-cli/package.json]: Not sure about this.

File: package.json (hunk 1 above)
Comment (irae) [inline package.json]: Yes, but I think we need to do it differently. I think we need to list all linked mendel-* modules and remove them. For instance, when switching from v2 to master we should be able to unlink the v2 modules without having them on the tree, right?

Comment (irae) [inline package.json]: Wait... what is "a"? confused, maybe you know better how npm works with links 😜

**Pull request merged**